### DISCUSSION TITLE
Tutorial on Property Chains

### DISCUSSION THREAD

**Discussion by shawntanzk** [jamesaoverton/obook#326] [Tutorial requests]
I realise im SUPER underknowledged (is that a word?) about property chains.
I've gone through a bit just from work, but I'm not confident to handle it myself.
Would be nice to have a tutorial on that -> conceptual, how its used, etc. 

Thanks :)

**Comment by sabrinatoro** [comment]
@sierra-moxon, I think this is the lesson you were also requesting. 
I think you can vote for this issue. (somewhere :-) )
